res/xml/cyrl_ueishsht.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="УЕИШЩ (Български, БДС)" script="cyrillic">
+  <row>
+    <key key0="у" key2="1" key4="esc"/>
+    <key key0="е" key1="~" key2="2" key3="\@"/>
+    <key key0="и" key1="!" key2="3" key3="\#" key4="ѝ"/>
+    <key key0="ш" key2="4" key3="$"/>
+    <key key0="щ" key2="5" key3="%"/>
+    <key key0="к" key2="6" key3="^"/>
+    <key key0="с" key2="7" key3="&amp;" key4="§"/>
+    <key key0="д" key2="8" key3="*" key4="№"/>
+    <key key0="з" key2="9" key3="(" key4=")"/>
+    <key key0="ц" key2="0" key3="f11_placeholder" key4="f12_placeholder"/>
+    <key key0="б" key2="€"/>
+  </row>
+  <row>
+    <key key0="ь" key1="tab" key2="`"/>
+    <key key0="я"/>
+    <key key0="а"/>
+    <key key0="о" key2="-" key3="_"/>
+    <key key0="ж" key2="=" key3="+"/>
+    <key key0="г" key4="}" key3="{"/>
+    <key key0="т" key2="~" key3="[" key4="]"/>
```

Comment:
`~` is present twice.

Reply:
Thanks for noting it! I've just fixed it.